libs/eo/certificates/feature-details/eo-certificate-details.component.ts

```diff
@@ -25,7 +25,7 @@ import { WattIconComponent } from '@energinet-datahub/watt/icon';
 
 import { EnergyUnitPipe, eoCertificatesRoutePath } from '@energinet-datahub/eo/shared/utilities';
 import { EoCertificate } from '@energinet-datahub/eo/certificates/domain';
-import { AibTechCode } from '@energinet-datahub/eo/metering-points/domain';
+import { TechCode } from '@energinet-datahub/eo/metering-points/domain';
```

Comment:
Tror ikke at de her skal have deres navn ændret. De var navngivet som de var for at specificere at vi følger aib standarden